build.gradle

```diff
@@ -67,48 +71,40 @@ sourceSets {
     }
 }
 
-pluginBundle {
-    website = 'https://github.com/gradle/android-cache-fix-gradle-plugin'
-    vcsUrl = 'https://github.com/gradle/android-cache-fix-gradle-plugin'
-
-    description = 'Gradle plugin to fix Android caching problems'
-
+gradlePlugin {
     plugins {
         androidCacheFixPlugin {
             id = 'org.gradle.android.cache-fix'
             displayName = 'Gradle Android cache fix plugin'
-            tags = ['android', 'cache', 'fix']
-            version = project.version
+            description = 'Gradle plugin to fix Android caching problems'
+            implementationClass = 'org.gradle.android.AndroidCacheFixPlugin'
```

Comment:
I think this means we can remove `src/main/resources/META-INF/gradle-plugins/org.gradle.android-cache-fix.properties` now, right?

Reply:
Good catch. Removed